web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/GetUserResourceLinksImpl.java

```diff
@@ -45,50 +44,66 @@ private ArrayList<ResourceLinkDTO> getAvailableFunctionsForUser(WebCertUser user
 
         if (hasAccessToSearchCreatePage(user)) {
             resourceLinks.add(
-                    ResourceLinkDTO.create(
-                            ResourceLinkTypeDTO.ACCESS_SEARCH_CREATE_PAGE,
-                            "Sök / skriv intyg",
-                            "",
-                            true
-                    )
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.ACCESS_SEARCH_CREATE_PAGE,
+                    "Sök / skriv intyg",
+                    "",
+                    true
+                )
             );
         }
 
         if (hasAccessToDraftList(user)) {
             resourceLinks.add(
-                    ResourceLinkDTO.create(
-                            ResourceLinkTypeDTO.ACCESS_DRAFT_LIST,
-                            "Ej signerade utkast",
-                            "",
-                            true
-                    )
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.ACCESS_DRAFT_LIST,
+                    "Ej signerade utkast",
+                    "",
+                    true
+                )
             );
         }
 
         if (hasAccessToSignedCertificatesList(user)) {
             resourceLinks.add(
-                    ResourceLinkDTO.create(
-                            ResourceLinkTypeDTO.ACCESS_SIGNED_CERTIFICATES_LIST,
-                            "Signerade intyg",
-                            "",
-                            true
-                    )
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.ACCESS_SIGNED_CERTIFICATES_LIST,
+                    "Signerade intyg",
+                    "",
+                    true
+                )
             );
         }
 
         if (isLogOutAvailable(user)) {
             resourceLinks.add(
-                    ResourceLinkDTO.create(
-                            ResourceLinkTypeDTO.LOG_OUT,
-                            "Logga ut",
-                            "",
-                            true
-                    )
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.LOG_OUT,
+                    "Logga ut",
+                    "",
+                    true
+                )
             );
         }
+
+        if (isChangeUnitAvailable(user)) {
+            resourceLinks.add(
+                ResourceLinkDTO.create(
+                    ResourceLinkTypeDTO.CHANGE_UNIT,
+                    "Byta vårdenhet",
```

Comment:
Shouldn't the text be "Byt vårdenhet"? Since the resource link will be be used to provide the frontend with the necessary text for the "Switch unit"-functionality maybe also some other text should be included? Since the functionality of "Switch unit" now is controlled by a resource link it could be a nice improvement to add a resource link for "Choose unit" which decides if the modal should be shown and provides the frontend with text.

Reply:
I agree. I changed to "Byt vårdenhet". I also created a new branch "INTYGFV-14453" and created the resource link for choose unit. Please have a look at that PR.